### PR TITLE
feature: add dates to guide

### PR DESCRIPTION
**Pull request by kamilfedio**
Okay, I added dates
 - last updated 
 - created
 to every guide on detail's page. I think extracting the newest date on modified and created from all questions faq's. If you think there is the better option such us dates under each faq let me know!

ss's:
image

### CODE REVIEW

File: lib/features/guide_detail_view/guide_detail_view.dart [hunk 1]
```diff
@@ -76,6 +78,27 @@ class _GuideDetailDataView extends ConsumerWidget {
...
             ),
+            SliverPadding(
+              padding: const EdgeInsets.only(
+                bottom: GuideDetailViewConfig.bottomPadding,
+              ),
+              sliver: SliverToBoxAdapter(
+                child: Column(
+                  children: [
+                    Text(
+                      "${context.localize.created_at} ${context.getTheLatesCreatedDateGuide(questions: value.questions) ?? context.localize.not_date_available}",
```
Comment (mikolaj-jalocha) [inline lib/features/guide_detail_view/guide_detail_view.dart:89]: When there's not date available I'd show just blank space

File: lib/features/guide_detail_view/guide_detail_view.dart [hunk 2]
```diff
@@ -76,6 +78,27 @@ class _GuideDetailDataView extends ConsumerWidget {
...
+                      style: context.textTheme.bodyGrey,
+                      textAlign: TextAlign.end,
+                    ),
+                    Text(
+                      "${context.localize.last_modified} ${context.getTheLatesUpdatedDateGuide(questions: value.questions) ?? context.localize.not_date_available}",
```
Comment (mikolaj-jalocha) [inline lib/features/guide_detail_view/guide_detail_view.dart:94]: Same here - blank space

File: lib/features/guide_detail_view/utils/get_the_latest_date.dart [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import "package:flutter/material.dart";
+import "package:intl/intl.dart";
+
+import "../repository/getGuideDetails.graphql.dart";
+
+extension GetTheLatestDateGuide on BuildContext {
```
Comment (mikolaj-jalocha) [inline lib/features/guide_detail_view/utils/get_the_latest_date.dart:6]: Please, check in some other place naming convention for extensions

File: lib/features/guide_detail_view/utils/get_the_latest_date.dart [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  String? getTheLatesCreatedDateGuide({
+    List<Query$GetGuideDetails$FAQ_Types_by_id$questions?>? questions,
+  }) {
+    final DateTime? newestDate = questions
+        ?.where((e) => e?.FAQ_id?.date_created != null)
+        .map((e) => e!.FAQ_id!.date_created!)
+        .reduce((a, b) => a.isAfter(b) ? a : b);
```
Comment (mikolaj-jalocha) [inline lib/features/guide_detail_view/utils/get_the_latest_date.dart:13]: I'm not sure, but I recall we have some extension that quite nicely filters nulls. It is called where not null or sth like that.
I'm not sure thought if it'd fit in this scenario but def worth checking ;)

File: lib/features/guide_detail_view/utils/get_the_latest_date.dart [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+    final DateFormat formatter = DateFormat("dd.MM.yyyy");
+    return newestDate != null ? formatter.format(newestDate) : null;
+  }
+
+  String? getTheLatesUpdatedDateGuide({
+    List<Query$GetGuideDetails$FAQ_Types_by_id$questions?>? questions,
+  }) {
+    final DateTime? newestDate = questions
```
Comment (mikolaj-jalocha) [inline lib/features/guide_detail_view/utils/get_the_latest_date.dart:22]: Same here.

File: lib/l10n/app_pl.arb [hunk 6]
```diff
@@ -113,5 +113,8 @@
         "example": "15"
       }
     }
-  }
+  },
+  "last_modified": "Ostatnia modyfikacja",
```
Comment (mikolaj-jalocha) [inline lib/l10n/app_pl.arb:117]: I'd say "Ostatnia aktualizacja"

**Review by mikolaj-jalocha — CHANGES_REQUESTED** [review]
First, I apologize for such a long reply time. 

Few minor suggestions of mine. 

In terms of where the update should be place, I thought about upper section of the article . In small print just below main cover? @simon-the-shark  @wkrzos, wdyt?  

At many eg. news sites such convention is adopted. Date of publication and date of actualization is displayed just below main photo in small print (presumably for users who might wonder if the article is worth reading mainly because of expiration :-) )

**Comment by kamilfedio** [comment]
Okay I changed but there is an error with code analysis and I can't find him :/

**Comment by mikolaj-jalocha** [comment]
> Okay I changed but there is an error with code analysis and I can't find him :/

Don't worry. Known issue

**Review by mikolaj-jalocha — APPROVED** [review]
LGTM!

**Pull request merged**